server/vehicle/main.lua

```diff
@@ -204,5 +206,24 @@ function Ox.GeneratePlate()
     end
 end
 
+---Creates a unique vehicle vin number.
+---@return string
+function Ox.GenerateVin()
+    local vin = table.create(17, 0)
+    while true do
+        ::start::
+        for i = 1, 17 do
+            vin[i] = math.random(0, 1) == 1 and string.char(math.random(65, 90)) or math.random(0, 9)
+        end
```

Comment:
We've talked about VIN severable times and wanted to implement it somewhat "realistically" where it follows a set structure and each character means something - manufacturer, model, production time (probably just creation time, rather).

If it's just for the sake of "uniqueness" then the id column already does that.

Reply:
Not only for uniqueness, but also for PD to be able to look up the VIN number on the vehicle, in case the license plate is missing from the vehicle. It serves as additional data to find the vehicle in the MDT.

Reply:
> It serves as additional data to find the vehicle in the MDT.
I mean that's what the vehicle id is for.

Reply:
> > It serves as additional data to find the vehicle in the MDT.
> 
> I mean that's what the vehicle id is for.

I know, but it looks weird when you search for a car in MDT by ID instead of VIN. I will try to come up with a good method of generating VINs